### PR TITLE
Added contribution guidelines, modifies make commands.

### PR DESCRIPTION
**Pull request by KaixuYang**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+Code Style
+^^^^^^^^^^
+
+Greykite follows the `flake8 <https://flake8.pycqa.org/en/latest/>`_ code style.
+Contributed code must pass flake8 check before it can be merged.
+
+Greykite follows the `numpydoc guide <https://numpydoc.readthedocs.io/en/latest/format.html>`_ for docstrings.
+For more details, you can also read `guide from lsst <https://developer.lsst.io/python/numpydoc.html>`_.
+
+Greykite documentation uses `sphinx <https://www.sphinx-doc.org/en/master/>`_
```
Comment (al-bert) [inline CONTRIBUTING.rst:38]:
```suggestion
Greykite uses `sphinx <https://www.sphinx-doc.org/en/master/>`_
```

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+and `sphinx-gallery <https://sphinx-gallery.github.io/stable/index.html>`_ to generate documentation pages.
+Please make sure the content are well rendered if you contributes to the documentation.
```
Comment (al-bert) [inline CONTRIBUTING.rst:40]:
```suggestion
Please make sure the content are well rendered if you contribute to the documentation.
```

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+Please make sure the code/docstring styles are the same with the other code in the same file/module before creating a pull request.
```
Comment (al-bert) [inline CONTRIBUTING.rst:42]:
```suggestion
Please make sure the code/docstring styles are the same as the other code in the same file/module before creating a pull request.
```

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+  Please keep a screenshot or a text file of the testing results. Please also document the new feature you developed by writing either a quickstart example (indicates the functionalities of the feature) or a tutorial example (an end-to-end example including the feature as core functionality).
+
+* If you contributed to the documentation, please make sure the sphinx-build completes without any error
+
+  .. code-block::
+
+    make docs
+
+  as well as that the generated documentation pages looks good. The generated docs are in docs/build/docs/greykite/html. Please take a full screenshot of the generated page with your browser.
```
Comment (al-bert) [inline CONTRIBUTING.rst:103]:
```suggestion
  and that the generated documentation pages look good. The generated docs are in docs/build/docs/greykite/html. Please take a full screenshot of the generated page with your browser.
```

File: docs/conf.py [hunk 5]
```diff
@@ -9,10 +9,10 @@
 import datetime
 import os
 import pathlib
-import warnings
+import sys
+sys.path.insert(0, os.path.abspath(os.path.join("..")))
```
Comment (al-bert) [inline docs/conf.py:13]: Add a comment to explain

Comment (KaixuYang) [inline docs/conf.py:13]:
```suggestion
sys.path.insert(0, os.path.abspath(os.path.join("..")))  # adds path to find greykite folder during doc build.
```

Comment (KaixuYang) [reply docs/conf.py]: Added a comment.

**Review by al-bert — APPROVED** [review]
LGTM!

**Pull request merged**